src/main/g8/src/main/scala/lib/$if(add_sql_orm.truthy)$sql-orm$endif$/services/ExampleService.scala

```diff
@@ -0,0 +1,86 @@
+package lib.sql_orm.services
```

Comment:
Nice job, in a next PR it could be interesting to be more generic in order to give pre-done things instead of just example ;)